server/routes/tools.py

```diff
@@ -20,6 +20,15 @@
 bp = flask.Blueprint("tools", __name__, url_prefix='/tools')
 
 
+@bp.route('/rich_search')
```

Comment:
Nit: this should go in the search module?

https://github.com/datacommonsorg/website/blob/master/server/routes/search.py

Reply:
I moved the route to search.py but kept `tools` as part of the path. Should I remove it? Should I also move all other files out of the tools directories?